File: Java/Booking/ReleaseTicketFromBasketResult.java
package LTD_API;

public class ReleaseTicketFromBasketResult
{
    private boolean success;
    private GetBasketContentResult getBasketContentResult;
    
    public boolean getSuccess()
    {
        return success;
    }
    public void setSuccess(boolean success)
    {
        this.success = success;
    }

    public GetBasketContentResult getGetBasketContentResult()
    {
        return getBasketContentResult;
    }
    public void setGetBasketContentResult(GetBasketContentResult getBasketContentResult)
    {
        this.getBasketContentResult = getBasketContentResult;
    }
}

File: PHP/Booking/ReleaseTicketFromBasketResult.php
<?php
namespace LTD_API;

class ReleaseTicketFromBasketResult
{
    private $success;
    //instance of GetBasketContentResult
    private $getBasketContentResult;
    
    public function getSuccess()
    {
        return $this->success;
    }
    public function setSuccess($success)
    {
        $this->success = $success;
    }

    public function getGetBasketContentResult()
    {
        return $this->getBasketContentResult;
    }
    public function setGetBasketContentResult($getBasketContentResult)
    {
        $this->getBasketContentResult = $getBasketContentResult;
    }
}

